Formula/jetty.rb

```diff
@@ -31,6 +50,17 @@ def install
     Dir.glob(libexec/"bin/*.sh") do |f|
       (bin/File.basename(f, ".sh")).write_env_script f, env
     end
+    return if Hardware::CPU.intel?
```

Comment:
We can. I originally didn't as resource isn't using the stable release of library. However, I don't think the code has changed since stable release other than the dependencies in Maven pom.xml.